### ISSUE TITLE
dhall-json bounds on Hackage

### ISSUE THREAD

**Issue by vmchale** [dhall-lang/dhall-haskell#1814]
Currently seem to be broken: see [matrix CI]

Thanks! :)

**Comment by sjakobi** [comment]
Can you give more details?

I see https://matrix.hackage.haskell.org/#/package/dhall-json/1.6.4/ghc-8.0.2@1589854941, but the error seems very mysterious:

``` 
src/Dhall/JSONToDhall.hs:361:1: error:
    Failed to load interface for ‘Data.Map.Merge.Lazy’
    It is a member of the hidden package ‘containers-0.6.2.1’.
    Perhaps you need to add ‘containers’ to the build-depends in your .cabal file.
    Use -v to see a list of the files searched for.
```

Can you reproduce that?

**Comment by Gabriel439** [comment]
@sjakobi: It means that `dhall-json` depends on `containers-0.6.1.1` or newer because of the use of `Data.Map.Merge.Lazy` but doesn't have a matching lower bound on its `containers` dependency

**Comment by sjakobi** [comment]
Ohhh! I was confused by the "hidden package" bit and the hint for the cabal file. Sorry!

Yeah, we should add a lower bound then. `Data.Map.Merge.Lazy` has been available since 0.5.9. I'll make a PR.
